pages/web-lit-basics-two.jsx

```diff
@@ -44,7 +44,7 @@ var curriculumList = [
         caption: "CC0 1.0 by Tungalik",
         subtitle: "Understanding Community Participation",
         description: "Learners will create and research survey questions about their community's Web use, learning about <strong>collaborating, community participation, and open practices</strong>.",
-        link: "https://chadsansing.github.io/curriculum-testing/web-lit-basics-two/session01-why-do-we-use-the-web.html"
+        link: "http://chadsansing.github.io/curriculum-testing/curriculum-migration/web-lit-basics-two-migrated/session01-why-do-we-use-the-web.html"
```

Comment:
Hey @hannahkane can you update the links so they are `https` instead of `http` :wink: